po/pt-br.po

```diff
@@ -0,0 +1,163 @@
+# Tradução para português brasileiro do spot.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```

Comment:
```suggestion
# Copyright (C) 2021 THE spot'S COPYRIGHT HOLDER
```
I _think_ this should look like this, see `en.po`:
https://github.com/xou816/spot/blob/108e27abddb5dbbcde24dfde001d39f192622e6c/po/en.po#L2